build.sbt

```diff
@@ -115,7 +116,22 @@ lazy val server = (project in file("server")) enablePlugins(JavaAppPackaging) se
   ),
   Compile / PB.targets := Seq(
     scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
-  )
+  ),
+  daemonUser in Docker := "root",
+  daemonUserUid in Docker := None,
+  // insert setcap to before entrypoint
+  dockerCommands := {
+    val commandLen = dockerCommands.value.length
+    (dockerCommands.value.take(commandLen-2)
+      ++ Seq(Cmd(
+                "RUN",
+                "setcap", """cap_net_bind_service=+eip""",
```

Comment:
I found that `setcap` actually didn't work for shell scripts. The reason your change is working is because we switch to the root user (Removing setcap and running using root still works). Looks like what we need is just:

```
daemonUser in Docker := "root",
daemonUserUid in Docker := None
```